### DISCUSSION TITLE
What does loading translation tables in Mirth connect mean?

### DISCUSSION THREAD

**Discussion by Poodeshwaran** [nextgenhealthcare/connect#5591] [General]
What does mean loading translation tables in Mirth connect?

**Comment by jonbartels** [comment]
What is the exact message you are seeing? When do you see this message?

**Comment by wi4seo** [comment]
Hello Everyone,

FREE Registration for Mirth Unconference 2023 is Now Opened!

To know more, please visit the event page and register yourself: https://wi4.org/mirth-connect-unconference-feb-2023/

Stay Tuned for More Updates!​

​Thanks
